react/Footer/FooterLink/FooterLink.js

```diff
@@ -36,9 +38,16 @@ FooterLink.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]),
-  linkRenderer: PropTypes.func.isRequired
+  linkRenderer: PropTypes.func.isRequired,
+  authedHref: PropTypes.string,
```

Comment:
@dborysiewicz since it seems we're removing the nudge parameter, is this still true?